pubsub/pubsub.go

```diff
@@ -22,16 +22,30 @@ import (
 
 // PubSub is the interface for message buses.
 type PubSub interface {
+	BulkMessager
```

Comment:
Per the conversation on the other PR, IMHO this should be done in the runtime and not via a stub method.